tools/make/lint.mk

```diff
@@ -64,3 +64,7 @@ lint: lint.shellcheck
 lint.shellcheck:
 	@echo Running Shellcheck linter ...
 	@shellcheck tools/hack/*.sh
+
+.PHONY: gen-check
+gen-check: generate manifests
+	tools/hack/gen-check.sh
```

Comment:
wdyt of skipping the script and adding the check in the Makefile itself ?
```
	@if [ ! -z "`git status --porcelain`" ]; then \
		@echo "ERROR: Some files need to be updated, please run 'make generate' and `make manifests` to include any changed files to your PR"; \
		git diff --exit-code; \
	fi
```

Reply:
it's ok for now, let's create a new shell file if there're too much lines in the future.